optimizely/client/client_test.go

```diff
@@ -1389,6 +1426,54 @@ func TestGetAllFeatureVariables(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestGetAllFeatureVariablesWithError(t *testing.T) {
+	testFeatureKey := "test_feature_key"
+	testVariableKey := "test_feature_flag_key"
+	testVariableValue := "teststring"
+	testUserContext := entities.UserContext{ID: "test_user_1"}
+	testVariationVariable := entities.VariationVariable{
+		ID:    "1",
+		Value: testVariableValue,
+	}
+	testVariable := entities.Variable{
+		DefaultValue: "defaultString",
+		ID:           "1",
+		Key:          testVariableKey,
+		Type:         entities.String,
+	}
+	testVariation := getTestVariationWithFeatureVariable(false, testVariationVariable)
```

Comment:
Can pass `true` as first arg instaed of setting it to true in the next line

Reply:
addressed